src/bessel.jl

```diff
@@ -758,3 +758,44 @@ External links: [DLMF](https://dlmf.nist.gov/10.2.6), [Wikipedia](https://en.wik
 See also: [`hankelh2(nu,x)`](@ref SpecialFunctions.hankelh2)
 """
 hankelh2x(nu, z) = besselhx(nu, 2, z)
+
+"""
+    fastabs(x)
+    
+In future just use `fastabs` from Base.Math
+https://github.com/JuliaLang/julia/blob/93fb785831dcfcc442f82fab8746f0244c5274ae/base/special/trig.jl#L1057
+"""
```

Comment:
This is an internal implementation detail, no need to put in a docstring.  I'd remove the docstring and just add a comment

Reply:
Sure, I removed it